doc/templates.rst

```diff
@@ -587,7 +587,9 @@ exist:
   string. They are useful whenever you need a string in the template (for
   example as arguments to function calls, filters or just to extend or include
   a template). A string can contain a delimiter if it is preceded by a
-  backslash (``\``) -- like in ``'It\'s good'``.
+  backslash (``\``) -- like in ``'It\'s good'``. If the string contains a
+  backslash (e.g. for PHP namespaces) escape it by doubling it
```

Comment:
I would remove the example in parenthesis and find another example as this document should be designer-friendly/non-PHP-focused as much as possible.